docs/sections/fundamentals/basics.md

```diff
@@ -249,65 +257,144 @@ As a final step, let's have a look at the provenance of this simple calculation.
 The provenance graph can be automatically generated using the verdi CLI.
 Let's generate the provenance graph for the `multiply` calculation function we have just run with PK = 3:
 
+:::{margin}
+**Remember:** the PK of the calculation node can be different for your database.
+:::
+
 ```{code-block} console
 
 $ verdi node graph generate 3
 
 ```
 
-```{note}
+The command will write the provenance graph to a `.pdf` file.
+The name of said file will start with the PK of your calculation node and have an intermediate `.dot` extension (in this case it would be `3.dot.pdf`).
+
+```{eval-rst}
+.. tabs::
+
+    .. tab:: Quantum Mobile
+
+        Opening files inside of the Quantum Mobile VM is easy!
+        All you need to do is open an explorer windows and go to the working directory in which you generated the file.
+        Then you can simply open it by double clicking on it.
+        Alternatively, you can open the file directly from the terminal by running:
+        
+        .. code-block:: console
 
-Remember that the PK of the calculation function can be different for your database.
+            $ evince 3.dot.pdf
+
+    .. tab:: AiiDAlab cluster
+
+        Opening files when working with the AiiDAlab cluster is easy!
+        All you need to do is go to the main page and open an explorer windows.
+        You can then just go to the working directory in which you generated the file and double click on it so that it will simply open in a new browser tab.
```

Comment:
I'm still not sure if this is even necessary or we can trust that they will now how to open a pdf inside a virtual machine with GUI and/or the jupyterhub interface...

Reply:
Well, we should show this in the beginning, really. But a reminder with a screen shot can't hurt?